procmgr/processmanager.cpp

```diff
@@ -11121,23 +11104,24 @@ int ProcessManager::mountDBRoot(std::string dbrootID)
     //send msg to ProcMon if not local module
     if ( config.moduleName() == moduleName )
     {
-        string cmd = "export LC_ALL=C;mount " + startup::StartUp::installDir() + "/data" + dbrootID + " > /tmp/mount.txt";
+		string tmpMount = tmpLogDir + "/mount.log";
+        string cmd = "export LC_ALL=C;mount " + startup::StartUp::installDir() + "/data" + dbrootID + " > " + tmpMount;
         system(cmd.c_str());
 
         if ( !rootUser)
         {
-            cmd = "sudo chown -R " + USER + ":" + USER + " " + startup::StartUp::installDir() + "/data" + dbrootID + " > /dev/null";
+            cmd = "chown -R " + USER + ":" + USER + " " + startup::StartUp::installDir() + "/data" + dbrootID + " > /dev/null";
```

Comment:
won't work as not-root, but also not necessary since it's probably already owned by the user that did the installation.